apps/codeforafrica/src/lib/api.netlify-cms/getSeo.js

```diff
@@ -0,0 +1,15 @@
+import { join } from "path";
+
+import getSettings from "./sections/getSettings";
+import { getCollectionBySlug } from "./utils";
+
+const indexPageDir = join(process.cwd(), "content/pages");
+
+export default function getSeo(page, fields = ["seo"]) {
+  const generalSettings = getSettings("general");
+  const defaultSeo = generalSettings?.seo;
+
+  const { seo } = getCollectionBySlug(indexPageDir, page, fields).items;
```

Comment:
1. `getCollectionBySlug` already path.join args, we don't need to do it twice.
2. why would `getSeo` allow `fields` to be changed?

```suggestion
export default function getSeo(page) {
  const generalSettings = getSettings("general");
  const defaultSeo = generalSettings?.seo;

  const { seo } = getCollectionBySlug("content/pages", page, ["seo"]).items;
```